Spinner/Spinner.module.less

```diff
@@ -0,0 +1,196 @@
+// Spinner.module.less
+//
+@import "../styles/mixins.less";
+@import "../styles/variables.less";
+@import "../styles/skin.less";
+
+@keyframes spinner {
```

Comment:
We'll likely want a more specific name here, since there may be multiple types of spinners.